src/test/java/org/openelisglobal/BaseWebContextSensitiveTest.java

```diff
@@ -76,25 +76,28 @@ protected void executeDataSetWithStateManagement(String datasetFileName) throws
         if (datasetFileName == null) {
             throw new NullPointerException("Please provide test dataset file to execute!");
         }
-
+    
         IDatabaseConnection connection = null;
         InputStream inputStream = null;
-
+    
         try {
             connection = new DatabaseConnection(dataSource.getConnection());
+    
+            // FIX: Use PostgreSQL-specific metadata handler
             DatabaseConfig config = connection.getConfig();
             config.setProperty(DatabaseConfig.FEATURE_ALLOW_EMPTY_FIELDS, true);
-
+            config.setProperty(DatabaseConfig.FEATURE_CASE_SENSITIVE_TABLE_NAMES, false);
+            config.setProperty(DatabaseConfig.PROPERTY_DATATYPE_FACTORY, new org.dbunit.ext.postgresql.PostgresqlDataTypeFactory());
```

Comment:
i dont think you need the changes in this class. unless you have a clear reason?